theNumber.js

```diff
@@ -0,0 +1,108 @@
+const readline = require('readline');
+const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+let guessCount = 0 //turn counter for guess
+
+
+function ask(questionText) {
+    return new Promise((resolve, reject) => {
+        readlineInterface.question(questionText, resolve);
+    });
+}
+// ********* Game begins below this line*************
+
+const min = 1;
+const max = 100;
+const acceptableNumbers = aceptNumb();
+
+let thePlayerNumber;
+//stores computer guesses in an array
+let alreadyGuessed = [];
+let compGuesses = comparingNumbers()
+// this function generates the computer's guesses
+function compGuesses(min, max) {
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+// this function starts the game by asking player name and giving them a nickname if they don't put anything in
+
+let name = gameName()
+async function gameName() {
+    name = await ask('What is your name? ')
+    if (name) {
+        console.log('Hello ' + caps(name) + '!')
+    } else {
+        console.log("Hmm, you don't appear to have a name.  I shall call you Peadar")
+    }
+    return name;
+   
+}
+
+//turning the name into a lowercase string
+function caps(wds) {
+    wds = wds.toString().trim().toLowerCase()
+    let firstLetter = wds[0].toUpperCase();
+    let rest = wds.slice(1);
+    return firstLetter + rest;
+}
```

Comment:
It looks like this function is actually returning a capitalized version of the word, rather than a lowercase string like is written in the comments